src/ripple/rpc/tests/AccountObjects_test.cpp

```diff
@@ -0,0 +1,250 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2016 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#include <ripple/protocol/JsonFields.h>
+#include <ripple/json/json_value.h>
+#include <ripple/json/to_string.h>
+#include <ripple/json/json_reader.h>
+#include <ripple/test/jtx.h>
+
+namespace ripple {
+namespace test {
+
+static char const* bobs_account_objects[] = {
```

Comment:
I guess we should decide (in general) whether we prefer to have our cpp tests use json strings for expected outcomes or if we prefer to explicitly check certain fields values. Whatever we decide is best is fine with me and it's not a rigid requirement, but just a good starting point when writing these tests. If you did direct field checks instead, some of the data in these json strings wouldn't even been seen in the test code because you can use things like bob.human() whenever the issuer ID is needed. Just something to consider.